practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Nice prac overall! I'm guessing you will link to the follow-up GNNs for RecSys prac?</p><p> </p><p>There are some further English mistakes that I spotted, but I gave up flagging them at some point. I recommend running it through a spell checker at some point, to catch the errant typos.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>